cogboard-webapp/src/components/EditBoard.js

```diff
@@ -15,26 +15,32 @@ const StyledCancelButton = styled(CancelButton)`
 const EditBoard = ({ closeDialog, id, ...initialFormValues }) => {
   const dispatch = useDispatch()
 
-  const handleSaveClick = (values) => () => {
+  const handleSaveClick = (values) => {
     dispatch(saveBoard({ id, ...values }));
     closeDialog();
   };
 
   return (
     <BoardForm
       {...initialFormValues}
-      renderActions={values =>
+      onSubmit={handleSaveClick}
+      boardId={id}
+      renderActions={() => (
         <>
           <Button
-            onClick={handleSaveClick(values)}
             color="primary"
             variant="contained"
+            type="submit"
+            data-cy="board-form-submit-button"
```

Comment:
I thought we have a convention component-name-element.

Reply:
Yes, but this element will be renderer in board form component and we have the same in add board, so to keep it simple I decide to use this name.